build.sbt

```
@@ -102,7 +102,7 @@ lazy val root = (project in file("."))
       version,
       scalaVersion,
       sbtVersion,
-      BuildInfoKey("sipiVersion", sipiVersion),
+      BuildInfoKey("sipiVersion", knoraSipiBaseImage),
```

Should be also the new name reflected in the info key?
```suggestion
      BuildInfoKey("knoraSipiBaseImageVersion", knoraSipiBaseImageVersion),
```